modelerfour/test/scenarios/a-playground/namer.yaml

```diff
@@ -60,13 +58,13 @@ schemas: !<!Schemas>
           value: YELLOW
           language:
             default:
-              name: YELLOW
+              name: Yellow
               description: ''
         - !<!ChoiceValue> 
           value: blacK
           language:
             default:
-              name: blacK
+              name: BlacK
```

Comment:
Interesting that this still has a capitalized 'K' at the end when the `YELLOW` got cased correctly?

Reply:
It's two words ...